docker/dev/Dockerfile

```diff
@@ -192,7 +202,16 @@ RUN $CMAKE --build $BASE_PATH/parflow/build
 
 USER root
 RUN $CMAKE --install $BASE_PATH/parflow/build --prefix /opt/parflow
+
+# Install the locally built version of pftools
 USER ubuntu
+RUN python3 -m pip install /home/ubuntu/parflow/build/pftools/python
+RUN cp -R /home/ubuntu/parflow/build/pftools/python/parflow/tools/ref ~/.local/lib/python3.8/site-packages/parflow/tools/ref
+
+RUN rm -R /home/ubuntu/parflow/src
+RUN mkdir /home/ubuntu/parflow/src
 
 ENV PARFLOW_DIR /opt/parflow/
 WORKDIR /opt/parflow/
+
+RUN echo 'alias rebuild="~/parflow/src/docker/dev/rebuild.sh"' >> ~/.bashrc
```

Comment:
Alias the build script so that the dev can simply type "rebuild" to trigger the incremental build instead of needing to know the scripts location